src/main/java/seedu/address/logic/parser/ExpenseCommandParser.java

```diff
@@ -37,8 +37,12 @@ public ExpenseCommand parse(String args) throws ParseException {
         if (amounts.size() != 1) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, ExpenseCommand.MESSAGE_USAGE));
         }
+        Amount amount = amounts.get(0);
+        if (amount.value == 0) {
```

Comment:
Should use `isValidAmount` instead to validate, currently `expense p/p1 e/0.001` passes.

Also should update `ExpenseCommandParserTest.java` to include the edge cases that were previously covered by `AmountTest.java`, i.e. `assertFalse(Amount.isValidAmount(0)); // Not positive` and `assertFalse(Amount.isValidAmount(0.004999)); // Rounds to 0`.

Reply:
ParserUtil catches those already.

Reply:
`ParserUtil` uses `isValidAmount` which allows amounts rounded to 0 to pass through